pipelines/runstatus.go

```diff
@@ -0,0 +1,33 @@
+package pipelines
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/common/commands"
+	status "github.com/jfrog/jfrog-cli-core/v2/pipelines/commands"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	clientlog "github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/urfave/cli"
+)
+
+// fetchLatestPipelineRunStatus fetch pipeline run status and filter from pipeline-name and branch flags
```

Comment:
fetch --> fetches
filter --> filters

Let's apply the above grammar correction in all descriptions add in this PR.